docs/source/hi/concepts/git_vs_http.md

```diff
@@ -0,0 +1,62 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
```

Comment:
Since this file is not translated and not referenced in the toctree, can you remove it from the PR?
(same for all files in `/concepts` and `/guides`)